### PR TITLE
ref(ui): Use Layouts for the issueList

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/components/layouts/thirds.tsx [hunk 1]
```diff
@@ -85,17 +85,21 @@ export const Title = styled('h1')`
  * Uses a horizontal layout in wide viewports to put space between
  * the headings and the actions container. In narrow viewports these elements
  * are stacked vertically.
+ *
+ * Use `noActionWrap` to disable wrapping if there are minimal actions.
  */
-export const Header = styled('div')`
+export const Header = styled('div')<{noActionWrap?: boolean}>`
```
Comment (scttcper) [inline static/app/components/layouts/thirds.tsx:91]: 👏 👏 👏

File: static/app/views/issueList/overview.tsx [hunk 2]
```diff
@@ -1023,87 +1023,92 @@ class IssueListOverview extends React.Component<Props, State> {
...
+                caption={tct('Showing [displayCount] issues', {
+                  displayCount,
+                })}
+                pageLinks={pageLinks}
+                onCursor={this.onCursorChange}
+              />
+            </Layout.Main>
+
             {/* Avoid rendering sidebar until first accessed */}
             {renderSidebar && (
```
Comment (scttcper) [inline static/app/views/issueList/overview.tsx:1100]: after closing the sidebar it just sits at the bottom of the page
image

**Comment by robinrendle** [comment]
🌮

File: static/app/views/issueList/overview.tsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/views/issueList/overview.tsx:1100]: lol. who put that there

File: static/app/components/layouts/thirds.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/components/layouts/thirds.tsx:91]: yeah it's a good one

**Review by scttcper — APPROVED** [review]
no animation 😢

File: static/app/views/issueList/header.tsx [hunk 3]
```diff
@@ -96,86 +96,81 @@ function IssueListHeader({
...
+    <Layout.Header noActionWrap>
+      <Layout.HeaderContent>
+        <StyledLayoutTitle>{t('Issues')}</StyledLayoutTitle>
+      </Layout.HeaderContent>
+      <Layout.HeaderActions>
+        <ButtonBar gap={1}>
+          <Button
+            size="small"
+            data-test-id="real-time"
+            title={t('%s real-time updates', realtimeActive ? t('Pause') : t('Enable'))}
```
Comment (priscilawebdev) [inline static/app/views/issueList/header.tsx:108]: wouldn't it be better if we had the entire translation in one sentence? maybe the current form is not ideal when translating the sentence into other languages 🤷‍♀️

```suggestion
            title={realtimeActive ? t('Pause real-time updates') : t('Enable real-time updates')}
```

**Comment by evanpurkhiser** [comment]
Sorry scott. Removed the animation. Asserting my executive designer opinions in this PR

File: static/app/views/issueList/header.tsx (hunk 3 above)
Comment (evanpurkhiser) [reply static/app/views/issueList/header.tsx:108]: Yeah probably. I'll fix in a follow up since I just copied this over.

**Pull request merged**